app/views/home/user_profile_template.html.erb

```diff
@@ -79,7 +83,9 @@
                 </ul>
               </div>
             </div>
-            <div class="permissions">shared</div>
+            <div class="thread-parameters">
+               <div class="permissions">shared</div>
+            </div>
```

Comment:
> was there a reason the original designs were kept as part of this repository

Because the design has never actually been fully implemented yet! That said, I am trying to get round to commissioning a fresh new design that mops up piles of usability issues.